R/spaces_acl_put.R

```diff
@@ -0,0 +1,58 @@
+#' Set an Object's Access Control List (ACL)
+#'
+#' @param object (character) The Object to set the ACL on
+#' @param space (character) The Space the Object is found in
+#' @param body (character) The XML-formatted ACL. Can optionally be an
+#' \code{xml_document} if the \code{xml2} package is installed.
+#' @template spaces_args
+#' @param ... Additional argument passed to \code{\link[aws.s3]{put_acl}}
+#'
+#' @return \code{TRUE} or \code{FALSE} depending on whether the ACL was successfully set
+#' @export
+#'
+#' @references \url{https://developers.digitalocean.com/documentation/
+#' spaces/#acls}
+#' @references \url{https://developers.digitalocean.com/documentation/
+#' spaces/#set-object-acls}
+#'
+#' @examples
+#' \dontrun{
+#' # First, create a Space and upload an Object to it
+#' space_create("my-space")
+#' spaces_object_put("some-file.txt", space = "my-space")
+#'
+#' # You can get a copy of the ACL
+#' acl <- spaces_acl_get("some-file.txt", "my-space") %>% read_xml()
+#'
+#' # Modify it as needed, then update it
+#' spaces_acl_put("my-object", "my-space", acl)
```

Comment:
is `my-object` supposed to be equivalent to `some-file.txt` ?  if so could you change that, so peole can actually do the eg

Reply:
That was a mistake. Fixed.